paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/content/EmbeddedContentHelper.kt

```diff
@@ -207,7 +207,9 @@ internal class DefaultEmbeddedContentHelper @Inject constructor(
                 } else {
                     true
                 }
-            }
+            },
+            linkRowClicked = ::setSelectionAfterUserClick,
```

Comment:
Right now you have 4 callbacks that each call the same method, can you replace this with a single callback?